components/fxa-client/uniffi.toml

```diff
@@ -3,4 +3,6 @@ package_name = "mozilla.appservices.fxaclient"
 cdylib_name = "megazord"
 
 [bindings.swift]
+ffi_module_name = "MozillaRustComponents"
+ffi_module_filename = "fxa_clientFFI"
```

Comment:
Not sure, but do we still want to keep this if we're not adding the `fxa_client` yet? (until we can get NSS built for M1)

Reply:
Actually never mind, this more of a `uniffi` thing that it is a `Swift Packages` thing right?

Reply:
It won't do any harm here if we leave it, but also it's only required because of how we do the XCFramework build, so it might make sense to take it out just to avoid confusion.

Reply:
(I'm going to go ahead and remove it to avoid confusion)